server/src/controllers/files.controller.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Controller,
+  Get,
+  Param,
+  UseInterceptors,
+  ClassSerializerInterceptor,
+  Res,
+  StreamableFile,
+  Patch,
+  Body,
+  UseGuards,
+} from '@nestjs/common';
+import { Readable } from 'stream';
+import { Response } from 'express';
+import { FilesService } from '../services/files.service';
+import { UpdateFileDto } from '../dto/update-file.dto';
+import { JwtAuthGuard } from 'src/modules/auth/jwt-auth.guard';
+
+@Controller('files')
+@UseInterceptors(ClassSerializerInterceptor)
+export class FilesController {
```

Comment:
If we aren't using `#findAll` and `#update`, could we remove them?

Reply:
Also, let's try to stick with current naming conventions ie #index, #show etc when possible within a controller?

Reply:
Cool, I have removed the unused endpoints and renamed the fn